packages/graphql/graphql/types/Package.ts

```diff
@@ -88,7 +89,7 @@ export const Package = objectType({
 
     t.nonNull.list.field('dependencies', {
       type: Package,
-      resolve(me, __, ctx) {
+      resolve(me, __, ctx: Context) {
```

Comment:
ctx should already have the type of context, you just need to codegen into this package just like the UI package had been doing with `build:nexus`